ui/components/User.js

```diff
@@ -67,34 +71,29 @@ const User = (props) => {
   };
 
   const handleGetToken = () => {
-    dataFetch(
-      '/api/token',
-      { credentials: 'same-origin' },
-      (data) => {
+    triggerGetToken()
+      .unwrap()
+      .then((data) => {
         exportToJsonFile(data, 'auth.json');
-      },
-      (error) => ({ error }),
-    );
+      })
+      .catch((error) => {
+        console.error('Error fetching token through useLazyGetTokenQuery() hook: ', error);
```

Comment:
provide a proper and meaningful error message to user.
Use error property of rtk to get the actual error message.
```suggestion
        console.error('Error while fetching token ', error);
```

Reply:
@sudhanshutech, in this case, should we be letting the user know of this error or no?

Reply:
yes it is good to show user a error here because from user side also there can be reason of error while fetching the token and having issues in logging. But we must use more exact info of error which comes from rtk itself

Reply:
Ohh Lee already mentioned my concern here @dottharun, ground here is that it is always a good idea to let user know the state of UI, in this by showing this error in toaster for which you can use `notify()` hook, however, if error if not blocking UI and can be avoided should go on console, I hope this clarifies the ground here